src/components/PageSections/Footer/_Footer.scss

```diff
@@ -4,6 +4,7 @@
 }
 
 .coa-Footer {
+  margin-top: 8rem;
```

Comment:
80 Pixels as requested in issue [#2009](https://app.zenhub.com/workspaces/office-of-design-and-delivery-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/2009)